scripts/rctGCS.py

```diff
@@ -415,12 +417,14 @@ def __createWidgets(self):
                text="Connect", command=self.__handleConnectInput).grid(column=0, row=0, sticky='new')
 
         # COMPONENTS TAB
-        frm_components = CollapseFrame(frm_sideControl, 'Components')
-        frm_components.grid(column=0, row=1, sticky='new')
+        self.statusWidget = StatusDisplay(frm_sideControl, self)
+        self.statusWidget.grid(column=0, row=1, sticky='new')
 
+        '''
         lbl_componentNotif = tk.Label(
-            frm_components.frame, width=self.SBWidth, text='Vehicle not connected')
+            self.statusWidget.frame, width=self.SBWidth, text='Vehicle not connected')
         lbl_componentNotif.grid(column=0, row=0, sticky='new')
```

Comment:
I believe this was old code, currently commented out, that got changed because I replaced every instance of the old components frame with the one I created.